app/controllers/orga/conferences_controller.rb

```diff
@@ -1,9 +1,14 @@
 class Orga::ConferencesController < Orga::BaseController
 
+  def new
+    @conference = Conference.new
+  end
+  
   def create
-    conference.season = current_season
+    @conference = Conference.new(conference_params)
```

Comment:
This may come down to personal preference (+ I don't know if this still works with Rails 5.1), but I usually try to have all params in place before initializing:
```ruby
@conference = Conference.new(conference_params.merge(season: current_season)
```